qa/rspec/commands.rb

```diff
@@ -50,10 +52,16 @@ def to_s
 
   class CommandsFactory
 
-    def self.fetch(type)
+    def self.fetch(type, host)
       case type
       when "debian"
-        return DebianCommands.new
+        if host.start_with?("ubuntu")
```

Comment:
actually ubuntu is debian based, so most of the interaction will be similar, but not the same for some outputs. This is the main reason. 

On the other side, you can see the opensuse example, is redhat (rpm) based, but the hole interaction is completely different, so I decided to have a different type. 

Makes sense to you?